### ISSUE TITLE
Mac ocascript can likely be modified for correct string output

### ISSUE THREAD

**Issue by danielpclark** [danielpclark/clock_window#9]
The last line of the ocascript `return {frontAppName, windowTitle}` can likely be changed to return the desired text we want.  If we swap the two variables they'll be in the right order.  It would be nice if the ocascript would also join the two variables into a string joined with a dash " - "

[lib/clock_window/oscommand.rb#L45](https://github.com/danielpclark/clock_window/blob/master/lib/clock_window/oscommand.rb#L45)

Once this is done we can change

``` ruby
format = Filters.new(
  substitutions: [ [Regexp.new(/([^,]*)[, ]{1,3}(.*)/), '\2 - \1'] ].
    add_if(@filter_opts.has_key? :substitutions){@filter_opts.delete(:substitutions)},
  **@filter_opts
)
```

to simply

``` ruby
format = Filters.new(**@filter_opts)
```

at lines [lib/clock_window/oscommand.rb#L49-L53](https://github.com/danielpclark/clock_window/blob/master/lib/clock_window/oscommand.rb#L49-L53)

Once this is done the test's will need to be rewritten for the Mac.  They'll basically be `format.call(x) == x` once formatting is no longer needed.

**Comment by danielpclark** [comment]
Just need to update the tests for this.

**Comment by danielpclark** [comment]
Done! https://github.com/danielpclark/clock_window/commit/45ceac1dcbd8b6b4eb722f6dc3296a0625763c38
